### PR TITLE
Private regsitry responsibility model

### PR DESCRIPTION
**Pull request by mhurtrel**
first version

### CODE REVIEW

**Review by H4516022Y — COMMENTED** [review]
review done with all stakeholders

**Comment by H4516022Y** [comment]
review done with all stakeholders
thanks for your review guides' team

File: pages/platform/private-registry/responsibility-model/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Import images using any Docker images or chart helm within OCI format | RA |  |
+
+#### 2.3. Customer Information System setup
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Deploy workloads using his images| RA |  |
+| Modify templates, helm charts, docketfile, urls for images' names, image dockers, etc . | RA |  |
```
Comment (OVH-Team-Guides) [inline pages/platform/private-registry/responsibility-model/guide.en-gb.md]:
```suggestion
| Modify templates, helm charts, dockerfile, urls for images' names, image dockers, etc . | RA |  |
```

File: pages/platform/private-registry/responsibility-model/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+### 4. Reverting
+
+#### 4.1. Reversibility model
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Schedule reversibility operations | RA | I |
+| Choose fallback infrastructures | RA |  |
+| Use Docker to transfer image by image or replicate images by using Harbor registry | RA |  |
```
Comment (OVH-Team-Guides) [inline pages/platform/private-registry/responsibility-model/guide.en-gb.md]:
```suggestion
| Use Docker to transfer image by image or replicated images by using Harbor registry | RA |  |
```

**Review by OVH-Team-Guides — COMMENTED** [review]
Hello @mhurtrel , @LisaPerrier , @H4516022Y , could you please check these 2 suggestions ? Thanks

File: pages/platform/private-registry/responsibility-model/guide.en-gb.md (hunk 2 above)
Comment (H4516022Y) [reply pages/platform/private-registry/responsibility-model/guide.en-gb.md]: No, the activity is : Use Docker to transfert image by image or to replicate images by Horador Registry

Comment (H4516022Y) [reply pages/platform/private-registry/responsibility-model/guide.en-gb.md]: could you correct it please ? 
Thank you

Comment (OVH-Team-Guides) [reply pages/platform/private-registry/responsibility-model/guide.en-gb.md]: It's been edited

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @mhurtrel

**Pull request merged**